### PR TITLE
i18n(es): Update of `layouts.md`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description. Update and translated content of `layouts.md` based on these commits 716cd0deaa33e1af87a55421dd061f5d7e24f00a & 6191c81493fdd0232140f052af1a8d4ffa70123e

### CODE REVIEW

File: src/pages/es/core-concepts/layouts.md [hunk 1]
```diff
@@ -5,104 +5,260 @@ description: Introducción a plantillas, un tipo de componente de Astro que se c
...
+  /** Disponible solo en Markdown */
+  rawContent: () => "# Astro 0.18 Release\nA little over a month ago, the first public beta [...]",
+  compiledContent: () => "<h1>Astro 0.18 Release</h1>\n<p>A little over a month ago, the first public beta [...]</p>",
+}
+```
+
+:::note
+Una plantilla Markdown/MDX tendrá acceso a todas sus [propiedades exportadas](/es/guides/markdown-content/#propiedades-exportadas) de archivo desde `Astro.props` **con algunas diferencias clave:**
+
+*   Encabezado información (es decir, elementos `h1 -> h6`) está disponible a través de la matriz `headings`, en lugar de una función `getHeadings()`.
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/layouts.md:186]:
```suggestion
*   Información de encabezados (es decir, elementos `h1 -> h6`) están disponibles a través del array `headings`, en lugar de la función `getHeadings()`.
```

File: src/pages/es/core-concepts/layouts.md [hunk 2]
```diff
@@ -5,104 +5,260 @@ description: Introducción a plantillas, un tipo de componente de Astro que se c
...
+
+*   `file` y `url` *también* están disponibles como propiedades anidadas de `frontmatter` (es decir, `frontmatter.url` y `frontmatter.file`).
+
+*   Valores definidos fuera del frontmatter (por ejemplo, declaraciones `export` en MDX) no están disponibles. Considera [importar una plantilla](#importando-plantillas-manualmente-mdx) en su lugar.
+:::
+
+### Importando Plantillas Manualmente (MDX)
+
+Tu puedes necesitar pasar información a tu plantilla MDX que no (o no puede) existir en tu frontmatter. En este caso, en su lugar, puedes importar y usar un [componente `<Layout />`](/es/core-concepts/layouts/) y pasarle propiedades como cualquier otro componente:
```
Comment (kevinzunigacuellar) [inline src/pages/es/core-concepts/layouts.md:195]:
```suggestion
Puedes necesitar pasar información a tu plantilla MDX que no (o no puede) existir en tu frontmatter. En este caso, en su lugar, puedes importar y usar un [componente `<Layout />`](/es/core-concepts/layouts/) y pasarle propiedades como cualquier otro componente:
```

**Review by kevinzunigacuellar — COMMENTED** [review]
Gracias por la PR @dreyfus92! Deje unos comentarios

**Comment by dreyfus92** [comment]
Muchas gracias por el review @kevinzunigacuellar 💚

**Pull request merged**